tidb-cloud/connected-ai-chat-in-im.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Connected: AI chat in IM
+summary: Introduces detailed information about the AI chat in IM
+---
+
+# Connected: AI chat in IM
+
+The AI chat in IM powered by PingCAP, is an offering to invite TiDB AI assistant chatbot to specific instant message (IM) channels for preliminary technical support and consultation. This service is based on Graph RAG built on top of TiDB Serverless Vector Storage.
+
+
+> **Limitation:**
+>
+> The Lark version is not fully ready yet. If you would like to take a try with the private preview release, please contact our support team at support@pingcap.com or your Technical Account Manager (TAM).
```

Comment:
While the Slack version is fully ready, the Lark version is still in private preview. If you're interested in trying the current Lark release, please contact our support team at support@pingcap.com or reach out to your Technical Account Manager (TAM).